internal/crossdock/server/oneway/server.go

```diff
@@ -103,6 +114,12 @@ func useCherami() bool {
 	return os.Getenv("CHERAMI") == "enabled"
 }
 
+// useGrpc checks to see if a grpc server is expected to be
+// available
+func useGrpc() bool {
+	return os.Getenv("GRPC") == "enabled"
```

Comment:
This is something I don't like either, but it's how the existing setup is. I agree with you, I think this could lead to misleading errors, and that we should fail if a transport is not able to come up, without having any conditionals in environment variables. I'm making a tracking issue for this as a cleanup task https://github.com/yarpc/yarpc-go/issues/916.